keyboards/mechlovin/delphine/rgb_led/readme.md

```diff
@@ -0,0 +1,15 @@
+# rgb_led
+
+![rgb_led](imgur.com image replace me!)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [Team Mechlovin'](https://github.com/yourusername)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
```

Comment:
This should be filled out, or the file deleted. (You already have a readme at `keyboards/mechlovin/delphine/readme.md`.)